Dockerfile

```diff
@@ -32,15 +33,13 @@ RUN apt-get update \
      git \
      google-chrome-stable \
      libmagic1 \
-     libpq-dev \
+     libpq5 \
      libxml2 \
      libxmlsec1 \
      libxmlsec1-openssl \
      make \
   && rm -rf /var/lib/apt/lists/* /src/*.deb
 # build-essential allows uv to build dependencies; increases image size by 240 MB
-# libpq-dev is for make-requirements-test.sh; increases image size by ~20 MB
-# libpq-dev can be replaced with libpq5 if pip-tools is replaced with uv in make-requirements-test.sh
```

Comment:
I was trying to understand this comment but I can't find `make-requirements-test.sh` or any reference to it other than here and https://github.com/dimagi/commcare-hq/pull/35420

Reply:
That was a typo. It was meant to reference [scripts/test-make-requirements.sh](https://github.com/dimagi/commcare-hq/pull/36391/files#diff-5e860f41be7c40cf999c4ca9d12d7983b6ba66e8eba75321ebcd9b50dd320a98)